tests/src/Interop/CMakeLists.txt

```diff
@@ -26,7 +26,10 @@ add_subdirectory(StringMarshalling/UTF8)
 add_subdirectory(MarshalAPI/FunctionPointer)
 add_subdirectory(MarshalAPI/IUnknown)
 add_subdirectory(SizeConst)
-add_subdirectory(ClassicCOM)
 add_subdirectory(DllImportAttribute/ExeFile)
 add_subdirectory(DllImportAttribute/FileNameContainDot)
 add_subdirectory(DllImportAttribute/Simple)
+
+if(WIN32)
```

Comment:
Does Win32 imply all Windows?  Or is there a corresponding WIN64 define else where in this file?

Reply:
That is a great question. I think Win32 is a reference to the Windows platform rather than anything to do with architecture, but I will confirm.